css/block.css

```diff
@@ -156,6 +158,84 @@ wb-context[closed] > wb-contains{
     overflow: hidden;
 }
 
+
+wb-value[type=number] > input{
```

Comment:
@JarrettSpiker I'll try to do this before my class starts!

The `background` rule implicitly sets `background-color` at the end. You can use the less specific rule to set all the defaults and the more specific rule to set a special attribute about it. But make sure you're covering all the bases in the general rule.